### DISCUSSION TITLE
Fundamental question

### DISCUSSION THREAD

**Discussion by budgetgerman** [Ifsttar/I-Simpa#326] [Q&A]
Hello everyone, I have a fundamental question, as I am not making progress with my research.

Does the software allow to calculate in a single project (or separately) the absorption and the acoustic insulation. I mean, to find out what percentage of absorption a material gives me towards the interior of the building. And on the other hand to obtain the decibels at a certain distance from the building, in an outdoor environment. Thank you very much

**Comment by Picaut** [comment]
Dear budgetgerman
Please check my answer to you issue #327 
Does it answer your question ?
Best regard.
J. Picaut

**Comment by budgetgerman** [comment]
Dear J Picaut
Good day, thank you very much for the prompt reply. It is in fact what I am
enquiring, and I have considered what you tell me. However, the issue I
find is that the SPL values measured by the receivers show me constant
values outside the building, regardless of how close or far they are
positioned from the sound sources.
I am not able to find out what the data input issue is or if it has to be
related to the fact that I am calculating under the reverberation theory
instead of SPPS.
Again thank you very much.
Best regards,

Germán A.

**Comment by budgetgerman** [comment]
Good day J Picaut
Once again, thank you very much for your prompt reply. I have been
reviewing your tutorials, and having considered them, I made again the
estimations, this time with SPPS method, but I still do not reach results I
can consider as valid:
- This time, the receivers that are outside the building, the SPL result is
constant: "-inf", regardless of the position or frequency of that receiver.
- On the other hand, when I attempt to perform a calculation considering a
sound source outside the building, the program execution fails and does not
generate any output.
Thank you very much and I look forward to your comments.
Best regards,

Germán A.
